run_tests.sh

```diff
@@ -47,4 +47,6 @@ golangci-lint run --disable-all --deadline=10m \
   --enable=vet \
   --enable=gosimple \
   --enable=unconvert \
-  --enable=ineffassign
+  --enable=ineffassign \
+  --enable=rowserrcheck \
```

Comment:
I just tested this and `rowserrcheck` doesn't actually work on our code. Probably because our calls to `db.Query` are in different funcs to our calls to `rows.Next()` and `rows.Err()`.

No harm adding it anyway. Might be useful someday in the future.